opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -85,11 +85,17 @@ type PVCSource struct {
 // ConfMgmt defines which additional services will be deployed
 type ConfMgmt struct {
 	AutoScaler  bool `json:"autoScaler,omitempty"`
-	Monitoring  bool `json:"monitoring,omitempty"`
 	VerUpdate   bool `json:"VerUpdate,omitempty"`
 	SmartScaler bool `json:"smartScaler,omitempty"`
 }
 
+type MonitoringStuck struct {
+	Enable         bool   `json:"enable,omitempty"`
+	MonitoringUser string `json:"monitoringUser,omitempty"`
+	Interval       string `json:"interval,omitempty"`
+	OfflinePlugin  string `json:"offlinePlugin,omitempty"`
```

Comment:
I would prefer this be named something like `PluginURL` to make it clearer whats inside. The offline part belongs in the description in the docs.